scene/gui/tree.cpp

```diff
@@ -1313,8 +1372,14 @@ int TreeItem::get_button_id(int p_column, int p_index) const {
 void TreeItem::erase_button(int p_column, int p_index) {
 	ERR_FAIL_INDEX(p_column, cells.size());
 	ERR_FAIL_INDEX(p_index, cells[p_column].buttons.size());
+	if (cells.write[p_column].buttons[p_index].accessibility_button_element.is_valid()) {
```

Comment:
```suggestion
	if (cells[p_column].buttons[p_index].accessibility_button_element.is_valid()) {
```
Checking this is `const`